talker_agent/install_agent.sh

```diff
@@ -1,17 +1,27 @@
-#!/bin/sh
+#!/bin/bash
 
 if [ "$DOWNLOAD_TALKER" = true ] ; then
+  # talker agent version can be:
+  #       tag            - for example: 1.8.2
+  #       tag-commit_id  - for example: 1.8.2-02ee1aed8275c79a5d0840505caaaef0fa53b26e
+  if [[ $TALKER_AGENT_VERSION == *"-"* ]]; then
+    IFS='-' read -r -a parts <<< "$TALKER_AGENT_VERSION"
+    commit_id=${parts[1]}
+  else
+    commit_id=v$TALKER_AGENT_VERSION
```

Comment:
no point in calling it `commit-id` if it's not a commit

Reply:
use `version` ?